src/io/flutter/view/MultiIconSimpleColoredComponent.java

```diff
@@ -582,19 +589,26 @@ private static boolean needFontFallback(Font font, String text) {
    */
   public int findFragmentAt(int x) {
     float curX = myIpad.left;
-    if (myIcon != null && !myIconOnTheRight) {
-      final int iconRight = myIcon.getIconWidth() + myIconTextGap;
-      if (x < iconRight) {
-        return FRAGMENT_ICON;
-      }
-      curX += iconRight;
-    }
 
     Font font = getBaseFont();
 
     int baseSize = font.getSize();
     boolean wasSmaller = false;
-    for (int i = 0; i < myAttributes.size(); i++) {
+    int i = 0;
+    int j = 0;
```

Comment:
`j` ==> `iconIndex`?

Reply:
I don't see where j is incremented.

Reply:
good catch. fixed this code block. The other similar code block is called.